catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/OrderRequestImpl.java

```diff
@@ -171,26 +175,38 @@ public State complete(DeliveryManifestHolder deliveryManifestHolder)
                 if (order.del_list != null) {
                     for (DeliveryDetails deliveryDetails : order.del_list) {
                         Destination destination = deliveryDetails.dests;
-                        FileLocation httpDestination = destination.f_dest();
-                        if (httpDestination != null) {
-                            String outputName = filename;
-
-                            if (StringUtils.isBlank(outputName)) {
-                                if (StringUtils.isNotBlank(httpDestination.file_name)) {
-                                    outputName = httpDestination.file_name;
-                                } else {
-                                    outputName = UUID.randomUUID()
-                                            .toString();
+
+                        try {
+                            FileLocation httpDestination = destination.f_dest();
+
+                            if (httpDestination != null) {
+                                String outputName = filename;
+
+                                if (StringUtils.isBlank(outputName)) {
+                                    if (StringUtils.isNotBlank(httpDestination.file_name)) {
+                                        outputName = httpDestination.file_name;
+                                    } else {
+                                        outputName = UUID.randomUUID()
+                                                .toString();
+                                    }
                                 }
+
+                                List<String> filesSent = writeData(httpDestination,
+                                        packageFormatType,
+                                        files,
+                                        outputName);
+                                PackageElement packageElement = new PackageElement();
+                                packageElement.files = filesSent.toArray(new String[filesSent.size()]);
+                                packageElements.add(packageElement);
                             }
+                        } catch (BAD_OPERATION e) {
+                        }
 
-                            List<String> filesSent = writeData(httpDestination,
-                                    packageFormatType,
-                                    files,
-                                    outputName);
-                            PackageElement packageElement = new PackageElement();
-                            packageElement.files = filesSent.toArray(new String[filesSent.size()]);
-                            packageElements.add(packageElement);
+                        try {
+                            String emailDest = destination.e_dest();
```

Comment:
Why create emailDest just for one line? I would just use destination.e_dest() in the method call for sendEmail.